packages/inventory/package.json

```diff
@@ -41,9 +41,11 @@
   "private": false,
   "dependencies": {
     "@opensrp/location-management": "^0.0.7",
-    "@opensrp/react-utils": "^0.0.2"
+    "@opensrp/react-utils": "^0.0.2",
+    "axios": "^0.21.1"
```

Comment:
what are we using axios for?

Reply:
to get access to the the progress event when uploading the file, the ui has an uploading status and validating status after file submission, fetch does not support this